kubernetes/helm/pinot/values.yaml

```diff
@@ -457,7 +469,10 @@ minionStateless:
     protocol: TCP
     name: minion
 
-  resources: {}
```

Comment:
can't comment below but we should also set zookeeper pod

Reply:
I don't know Zookeeper that well. How much memory/cpu would you recommend to use?

Reply:
I'm reading our chart values and it seems we have a section:
```
  ## Environmental variables to set in Zookeeper
  env:
    ## The JVM heap size to allocate to Zookeeper
    ZK_HEAP_SIZE: "256M"
```

but that env is being ignored by bitnami zookeeper chart and in fact 1024 is being used :facepalm:. So I guess I can include the same request specified to the rest of the services